### PR TITLE
CSYS-202: migrate radio

### PR DESCRIPTION
**Pull request by CavalcanteLucas**
**Link to task:**
https://labcodes.atlassian.net/browse/CSYS-202

**How to test:**
Download the repo, run the storyback and check that Radio story works as expected.

**Description of your solution:**
Added stories for Radio.

**Remarks:**
- Controls don't seem to have effect. Can't explain why.

### CODE REVIEW

**Review by lucianoratamero — CHANGES_REQUESTED** [review]
This PR doesn't follow the requirements of the task. Please review it and make the necessary changes.
And, please, mark me and @falconedani as reviewers when opening the PRs! :]

**Review by lucianoratamero — APPROVED** [review]
:rocket:

File: stories/Radio.stories.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      control: {
+        type: "select",
+        options: ["string", "number", "bool"]
+      }
+    }
+  }
+};
+
+export const Radio = (args) => (
+  <fieldset>
```
Comment (lucianoratamero) [inline stories/Radio.stories.js:19]: oh, wait, I approved, but you forgot to add a `<legend>` with a label here.

**Review by lucianoratamero — CHANGES_REQUESTED** [review]
sorry about that :x

File: stories/Radio.stories.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import { Radio as Component } from "../confetti-ds/src";
+
+export default {
+  title: "Radio/Radio",
+  component: Component,
+  argTypes: {
+    value: {
+      control: {
+        type: "select",
+        options: ["string", "number", "bool"]
```
Comment (lucianoratamero) [inline stories/Radio.stories.js:12]: these opptions are wrong. ["string", "number", "bool"] are **types** of values, not values. please remove this control.

File: stories/Radio.stories.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  }
+};
+
+export const Radio = (args) => (
+  <fieldset>
+    <Component
+      id="demo-radio-1"
+      name="demo-radio"
+      label="radio 1"
+      value="bool"
```
Comment (lucianoratamero) [inline stories/Radio.stories.js:24]: you are not passing the `args` as props to the components, so the controls from storybook aren't changing the components. please look at the Button example to see what I mean and add the args here.

**Review by lucianoratamero — CHANGES_REQUESTED** [review]

File: stories/Radio.stories.js (hunk 1 above)
Comment (CavalcanteLucas) [reply stories/Radio.stories.js:19]: fixed

File: stories/Radio.stories.js (hunk 2 above)
Comment (CavalcanteLucas) [reply stories/Radio.stories.js:12]: fixed

File: stories/Radio.stories.js (hunk 3 above)
Comment (CavalcanteLucas) [reply stories/Radio.stories.js:24]: thank you, fixed

**Comment by CavalcanteLucas** [comment]
All fixed were applied. Although the `ariaDisabled` field doesn't seem to be having the expected behaviour on the radio group. Yet the value of `ariaDisabled` still overrides the value of `disabled`. 

IMO a possibile solution would be to remove the `ariaDisabled` field from the story.

@falconedani What do you think about it?

**Review by lucianoratamero — APPROVED** [review]
:1st_place_medal:

**Pull request merged**